### ISSUE TITLE
Terraspace show --json adds newlines to json output

### ISSUE THREAD

**Issue by ddevaal** [boltops-tools/terraspace#126]
I use the command: ```TS_ENV=test terraspace show aks --plan aks.tfplan --json``` to produce a json file which I then parse with jq to show some statistics about the changes to be made in a merge request. However ```terraspace show --json``` adds several newlines to (it seems) random places in this json file. I tried to do the same in the cache/stack dir with ```terraform show``` and it does not add any newlines to random places.

Terraspace:
```
TS_ENV=test terraspace init aks && TS_ENV=test terraspace plan aks --out aks.tfplan && TS_ENV=test terraspace show aks --plan aks.tfplan --json | wc -l

3
```

Terraform (in the cache/stack directory):

```
terraform show -json aks.tfplan | wc -l

1 
```

I only expect one newline at the end of the file. Terraspace: **0.6.11,** Terraform: **1.0.2**

**Comment by drbjim** [comment]
you can create temporary file as json like below 
`TS_ENV=test terraspace output aks --format=json -o test.json`

**Comment by ddevaal** [comment]
> you can create temporary file as json like below
> `TS_ENV=test terraspace output aks --format=json -o test.json`

This doesn't produce anything for me.

**Comment by drbjim** [comment]
> > you can create temporary file as json like below
> > `TS_ENV=test terraspace output aks --format=json -o test.json`
> 
> This doesn't produce anything for me:
> 
> ```
> ❯ TS_ENV=test terraspace output aks --format=json
> Current directory: .terraspace-cache/westeurope/test/stacks/aks
> => terraform output -json
> {}
> ```
> 
> I guess this works _after_ something is created.

yes, this should be running after `terraform up`

**Comment by tongueroo** [comment]
Fix released in 0.6.15

**Comment by stempler** [comment]
> you can create temporary file as json like below `TS_ENV=test terraspace output aks --format=json -o test.json`

@drbjim How would one find out which arguments are supported here (e.g. that the `format` parameter is supported)? I did not find it in the CLI help or the documentation. Is there a specific place in the code you could point to where this is handled?
